packages/@ama-sdk/core/src/fwk/core/api-client.ts

```diff
@@ -51,8 +51,8 @@ export interface ApiClient {
   processFormData(data: any, type: string): FormData | string;
 
   /** Process HTTP call */
-  processCall<T>(url: string, options: RequestOptions, apiType: ApiTypes | string, apiName: string, reviver?: ReviverType<T> | undefined, operationId?: string): Promise<T>;
-
+  processCall<T>(url: string, options: RequestOptions, apiType: ApiTypes | string, apiName: string, reviver?: ReviverType<T> | undefined |
+    {[key: number]: ReviverType<T> | undefined}, operationId?: string): Promise<T>;
```

Comment:
`{[key: number]: ReviverType<T> | undefined}` or `Record<number, ReviverType<T> | undefined>` ?

Reply:
`{[responseCode: number]: ReviverType<T> | undefined}`
so it easier to understand what should be the key